src/pool/srv.c

```diff
@@ -72,11 +76,107 @@ fini(void)
 	return 0;
 }
 
+static int
+recreate_pool_layout()
+{
+	struct smd_pool_info    *pool_info = NULL, *tmp;
+	d_list_t                 pool_list;
+	char                     uuid_str[DAOS_UUID_STR_SIZE];
+	int			 rc = 0, pool_list_cnt, i, fd;
+	char			*path = NULL, *pool_dir = NULL;
+	uint64_t		 blob_sz;
```

Comment:
Just a note on the DAOS coding style: it is recommended to put each variable on its own line for readability, even when there are multiple variables of the same type.

Here is the C style guide, for reference: https://daosio.atlassian.net/wiki/spaces/DC/pages/4836655703/C+Style+Guide

Reply:
Corrected the code to follow the guidelines.